apps/client/src/features/app-settings/panel/feature-settings-panel/custom-fields/CustomFieldForm.tsx

```diff
@@ -66,28 +70,38 @@ export default function CustomFieldForm(props: CustomFieldsFormProps) {
 
   return (
     <form onSubmit={handleSubmit(setupSubmit)} className={style.fieldForm}>
-      <div className={style.column}>
-        <Panel.Description>Label (only alphanumeric characters are allowed)</Panel.Description>
-        {errors.label && <Panel.Error>{errors.label.message}</Panel.Error>}
-        <Input
-          {...register('label', {
-            required: { value: true, message: 'Required field' },
-            validate: (value) => {
-              if (value.trim().length === 0) return 'Required field';
-              if (!isAlphanumeric(value)) return 'Only alphanumeric characters are allowed';
-              return true;
-            },
-          })}
-          size='sm'
-          variant='ontime-filled'
-          autoComplete='off'
-        />
+      <div className={style.twoCols}>
+        <div>
+          <Panel.Description>Label (only alphanumeric characters are allowed)</Panel.Description>
+          {errors.label && <Panel.Error>{errors.label.message}</Panel.Error>}
+          <Input
+            {...register('label', {
+              required: { value: true, message: 'Required field' },
+              onChange: () => setValue('key', getValues('label').replaceAll(' ', '_')),
```

Comment:
forgot to use `customFieldLabelToKey` here
```suggestion
              onChange: () => setValue('key', customFieldLabelToKey(getValues('label'))),
```

Reply:
That's your own code so I'd say have at it?

Reply:
Yeah I just forgot to use it there 😉 
Just wanted to ask befor I changed stuff you were potentially working on